pocld/avahi_advertise.c

```diff
@@ -0,0 +1,267 @@
+/* avahi_advertise.c - part of pocl-daemon that performs mDNS on local network
```

Comment:
This is part of the server-side, so could have been C++ for more productive programming, but perhaps too late to convert.